api/README.md

```diff
@@ -710,3 +711,150 @@ The response has three top-level fields:
 }
 ```
 </details>
+
+## Test History
+
+### /api/history
+
+This endpoint accepts POST requests. It returns historical test run information for a given test name.
+
+#### JSON Request Payload
+```json
+{
+    "test_name": "example test name"
+}
+```
+#### JSON Response
+The returned JSON will contain a history of test runs for each major browser: Chrome, Firefox, Edge, and Safari.
+
+Each individual subtest run will have a `date`, `status`, and `run_id`.
+
+The first test entry for each browser is represented with an empty string. This represents the `Harness Status` if there are multiple tests, or the `Test Status` if there is only one test.
+
+<details><summary><b>Example JSON</b></summary>
+
+```json
+{
+    "results": {
+        "chrome": {
+            "": [
+                {
+                    "date": "2022-06-02T06:02:55.000Z",
+                    "status": "TIMEOUT",
+                    "run_id": "5074677897101312"
+                },
+                {
+                    "date": "2023-02-21T03:08:15.000Z",
+                    "status": "OK",
+                    "run_id": "5074677897101312"
+                },
+            ],
+            "subtest_name_1": [
+                {
+                    "date": "2022-06-02T06:02:55.000Z",
+                    "status": "PASS",
+                    "run_id": "5074677897101312"
+                },
+                {
+                    "date": "2022-08-14T06:02:55.000Z",
+                    "status": "FAIL",
+                    "run_id": "5074677897101312"
+                },
+            ],
+        },
+        "edge": {
+            "": [
+                {
+                    "date": "2022-06-02T06:02:55.000Z",
+                    "status": "OK",
+                    "run_id": "5074677897101312"
+                },
+                {
+                    "date": "2023-02-21T03:08:15.000Z",
+                    "status": "TIMEOUT",
+                    "run_id": "5074677897101312"
+                },
+            ],
+            "subtest_name_1": [
+                {
+                    "date": "2022-06-02T06:02:55.000Z",
+                    "status": "PASS",
+                    "run_id": "5074677897101312"
+                },
+                {
+                    "date": "2022-08-14T06:02:55.000Z",
+                    "status": "FAIL",
+                    "run_id": "5074677897101312"
+                },
+            ],
+            "subtest_name_2": [
+                {
+                    "date": "2022-06-02T06:02:55.000Z",
+                    "status": "TIMEOUT",
+                    "run_id": "5074677897101312"
+                },
+                {
+                    "date": "2022-09-25T23:49:35.000Z",
+                    "status": "PASS",
+                    "run_id": "5074677897101312"
+                },
+            ]
+        },
+        "firefox": {
```

Comment:
Might be too much data for documentation. Maybe keep Chrome & Firefox only? and with one subtest entry?

Reply:
I can cut it down some more 👍